iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/Converters.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm;
+
+import com.powsybl.commons.config.MapModuleConfig;
+import com.powsybl.commons.config.ModuleConfigUtil;
+import com.powsybl.iidm.parameters.Parameter;
+import com.powsybl.iidm.parameters.ParameterDefaultValueConfig;
+
+import java.util.Objects;
+import java.util.Properties;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public final class Converters {
```

Comment:
Converters is not a good name: Parameters seems better as this class converts nothing